src/org/usfirst/frc4904/robot/subsystems/GearIO.java

```diff
@@ -1,27 +1,28 @@
 package org.usfirst.frc4904.robot.subsystems;
 
 
-import org.usfirst.frc4904.robot.commands.OverridableCommand;
+import org.usfirst.frc4904.robot.commands.SetRampState;
 import org.usfirst.frc4904.standard.commands.Idle;
 import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
 import org.usfirst.frc4904.standard.subsystems.motor.Motor;
 import edu.wpi.first.wpilibj.DoubleSolenoid;
 import edu.wpi.first.wpilibj.command.Subsystem;
 
-public class GearIO extends Subsystem implements OverridableCommand {
+public class GearIO extends Subsystem {
 	protected final Motor intakeRoller;
 	protected final DoubleSolenoid gullWings;
 	protected final DoubleSolenoid ramp;
 	protected GearState currentState;
 	protected RampState currentRampState;
-	public boolean rampOverride;
 
 	public GearIO(Motor intakeRoller, DoubleSolenoid gullWings, DoubleSolenoid ramp) {
 		this.intakeRoller = intakeRoller;
 		this.gullWings = gullWings;
 		this.ramp = ramp;
 		setState(GearState.TRANSPORT);
-		setRampState(GearIO.RampState.EXTENDED);
+		if (new SetRampState(null).isNotOverridden()) {
+			setRampState(GearIO.RampState.EXTENDED);
```

Comment:
Let's just make the override on a command-by-command basis and not on a static level.